### PR TITLE
Fix typos/minor error in comments, and wrap some lines

### PR DESCRIPTION
**Pull request by daira**
This minimizes the diff in the example implementation of funding streams in ZIP 207.

Signed-off-by: Daira Hopwood <daira@jacaranda.org>

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -3944,15 +3945,16 @@ bool ContextualCheckBlock(
...
+                             REJECT_INVALID, "bad-cb-height");
         }
     }
 
     // Coinbase transaction must include an output sending 20% of
-    // the block reward to a founders reward script, until the last founders
-    // reward block is reached, with exception of the genesis block.
-    // The last founders reward block is defined as the block just before the
-    // first subsidy halving block, which occurs at halving_interval + slow_start_shift
+    // the block subsidy to a Founders' Reward script, until the last Founders'
```
Comment (oxarbitrage) [inline src/main.cpp]: `Founders' Reward` - Confusing for non English native speakers. 

Is it the accepted way to call it? i see it written different even inside the same ECC post for example: https://electriccoin.co/blog/electric-coin-company-q3-2019-transparency-report/

In the article, it refers to `Founders Reward` while in the tags at the bottom it is `#Founders' Reward`

Comment (str4d) [reply src/main.cpp]: Yes, `Founders' Reward` is the correct term, as used in the original blog post: https://electriccoin.co/blog/funding/

**Comment by str4d** [comment]
@zkbot r+

File: src/main.cpp (hunk 1 above)
Comment (oxarbitrage) [reply src/main.cpp]: Thanks.

**Pull request merged**